docs/source/intro.rst

```diff
@@ -1,6 +1,6 @@
-Intro 
-=====
+Introduction
+------------
+-Goal of the tool: To extend git-annex special remote implementation to Open Science Framework (OSF). This allows for OSF repository to be accessible via repository hosting services, and allows for direct access to OSF stored data via a repository hosting API, for example 'git clone' in Github.
+-What can I use this tool for?: The tool is most beneficial for easy access to data stored on OSF via Github. If you are sharing your dataset via OSF and code via Github, this will allow smooth integration of both along with unified version management provided by Datalab.
```

Comment:
Would suggest to avoid the `?:`  Maybe just delete the `:`.

Reply:
updated. please check.